UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json

```diff
@@ -0,0 +1,1799 @@
+                  "type": "display",
+                  "text": "on anticoagulants"
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "APO",
+      "text": "Accessment and Plan",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "ASM.1",
+          "text": "Assessment/Summary",
+          "type": "text"
+        },
+        {
+          "linkId": "PLAN.1",
+          "text": "Treatment Paln",
+          "type": "text"
+        },
+        {
+          "linkId": "ORD",
+          "text": "Orders",
+          "type": "group",
+          "item": [
+            {
+              "linkId": "ORD.1",
+              "text": "Medications",
+              "type": "text"
+            },
+            {
+              "linkId": "ORD.2",
+              "text": "Supplies",
+              "type": "text"
+            },
+            {
+              "linkId": "ORD.3",
+              "text": " Investigations / diagnostic testing",
+              "type": "text"
+            },
+            {
+              "linkId": "ORD.4",
+              "text": "Consults",
+              "type": "text"
+            },
+            {
+              "linkId": "ORD.5",
+              "text": "Other",
+              "type": "text"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "linkId": "SIGPNP",
+      "text": "Provider Signature",
+      "type": "group",
+      "item": [
+        {
+          "linkId": "SIGPNP.1",
+          "text": "Signature",
+          "type": "string",
+          "required": true
+        },
+        {
+          "linkId": "SIGPNP.2",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPractitionerInfoPrepopulation\".FullName"
+              }
+            }
+          ],
+          "text": "Name (Printed)",
+          "type": "string",
+          "required": true
+        },
+        {
+          "linkId": "SIGPNP.3",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPractitionerInfoPrepopulation\".Today"
+              }
+            }
+          ],
+          "text": "Date",
+          "type": "date",
+          "required": true
+        },
+        {
+          "linkId": "SIGPNP.4",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/cqf-expression",
+              "valueExpression": {
+                "language": "text/cql",
+                "expression": "\"BasicPractitionerInfoPrepopulation\".NPI"
+              }
+            }
+          ],
+          "text": "NPI",
+          "type": "string",
```

Comment:
Name and NPI are not populated. 

![Screen Shot 2020-05-06 at 9 49 44 AM](https://user-images.githubusercontent.com/58572522/81185035-17bb7080-8f7f-11ea-9f24-67ff44ec14de.png)